packages/x-adapter/package.json

```diff
@@ -36,11 +36,10 @@
   },
   "dependencies": {
     "@empathyco/x-deep-merge": "^1.3.0-alpha.25",
-    "@empathyco/x-get-safe-property-chain": "^1.3.0-alpha.4",
+    "@empathyco/x-utils": "^1.0.0-alpha.11",
```

Comment:
Should we deprecate the get-safe-property-chain package in npm?

Reply:
yes, I will do it when we merge this PR



https://docs.npmjs.com/deprecating-and-undeprecating-packages-or-package-versions